utils/src/test/java/io/ona/kujaku/helpers/WmtsCapabilitiesSerializerTest.java

```diff
@@ -0,0 +1,54 @@
+package io.ona.kujaku.helpers;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.io.InputStreamReader;
+
+import io.ona.kujaku.utils.BuildConfig;
+import io.ona.kujaku.utils.helpers.WmtsCapabilitiesSerializer;
+import io.ona.kujaku.utils.wmts.model.WmtsCapabilities;
+
+/**
+ *
+ * Created by Emmanuel Otin - eo@novel-t.ch 12/04/18.
+ */
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, manifest = Config.NONE)
+public class WmtsCapabilitiesSerializerTest {
+
+    private WmtsCapabilities getCapabilities() {
+
+        WmtsCapabilities capabilities = null ;
+
+        try {
+            InputStreamReader streamReader = new InputStreamReader(this.getClass().getClassLoader().getResourceAsStream("Capabilities.xml"));
+            WmtsCapabilitiesSerializer serializer = new WmtsCapabilitiesSerializer();
+            capabilities = serializer.read(WmtsCapabilities.class, streamReader, false);
+
+        } catch (Exception ex) {
```

Comment:
Declaring the exception in the method signature will make the test fail in case the exception happens which is still OK and much more traceable than assuming the exception and failing the test because the assert failed. What do you think?